### PR TITLE
switch to new style http return helpers

### PR DESCRIPTION
**Pull request by kfedor**

### CODE REVIEW

File: cmd/eskimo/users.go [hunk 1]
```diff
@@ -113,15 +103,7 @@ func (s *service) GetUserByUsername(ctx context.Context, r server.ParsedRequest)
...
-			m := fmt.Sprintf("user with username `%v` was not found.", req.Username)
-
-			return server.Response{
-				Code: http.StatusNotFound,
-				Data: server.ErrorResponse{
-					Error: m,
-					Code:  userNotFoundCode,
-				}.Fail(errors.Wrapf(err, "failed to get user by username")),
-			}
+			return *server.NotFound(err, userNotFoundCode)
```
Comment (victor-oancea-leftclick-io) [inline cmd/eskimo/users.go:106]: wrap err and add info from 124 & 116 to it

File: cmd/eskimo/users.go [hunk 2]
```diff
@@ -42,15 +40,7 @@ func (s *service) GetUserByID(ctx context.Context, r server.ParsedRequest) serve
...
-			m := fmt.Sprintf("user with id `%v` was not found.", req.ID)
-
-			return server.Response{
-				Code: http.StatusNotFound,
-				Data: server.ErrorResponse{
-					Error: m,
-					Code:  userNotFoundCode,
-				}.Fail(errors.Wrapf(err, m)),
-			}
+			return *server.NotFound(err, userNotFoundCode)
```
Comment (victor-oancea-leftclick-io) [inline cmd/eskimo/users.go:43]: wrap err and add info from 45 to it

Comment (kfedor) [reply cmd/eskimo/users.go:43]: done

File: cmd/eskimo/users.go (hunk 1 above)
Comment (kfedor) [reply cmd/eskimo/users.go:106]: done

**Review by victor-oancea-leftclick-io — APPROVED** [review]

**Review by nusanovkornilov-leftclick — APPROVED** [review]

**Pull request merged**